### PR TITLE
Epics.data size.fix

### PR DESCRIPTION
**Pull request by Mark-Booth**

### CODE REVIEW

File: org.eclipse.scanning.connector.epicsv3/src/org/eclipse/scanning/connector/epicsv3/EpicsV3DynamicDatasetConnector.java [hunk 1]
```diff
@@ -76,25 +78,27 @@
 
 	LinkedList<IDataListener> listeners = new LinkedList<>();
 	
+	int dim0,
+		dim1,
```
Comment (gerring) [inline org.eclipse.scanning.connector.epicsv3/src/org/eclipse/scanning/connector/epicsv3/EpicsV3DynamicDatasetConnector.java]: MINOR Declare "dim1" on a separate line. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1659)

File: org.eclipse.scanning.connector.epicsv3/src/org/eclipse/scanning/connector/epicsv3/EpicsV3DynamicDatasetConnector.java [hunk 2]
```diff
@@ -76,25 +78,27 @@
...
+		dim2;
```
Comment (gerring) [inline org.eclipse.scanning.connector.epicsv3/src/org/eclipse/scanning/connector/epicsv3/EpicsV3DynamicDatasetConnector.java]: MINOR Declare "dim2" on a separate line. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1659)

**Comment by gerring** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Comment by Mark-Booth** [comment]
This is a new pull request due to the rebase, but it was approved pre-rebase:

> MattTaylorDLS approved these changes a day ago
> 
> This looks fine to me

**Pull request merged**